plugins/generic/usageStats/js/UsageStatsFrontendHandler.js

```diff
@@ -123,15 +123,31 @@
 		graphData = pkpUsageStats.data[objectType][objectId];
 
 		// Turn the data set into an array
-		var dataArray = [], currentYear = new Date().getFullYear();
-		for(month in graphData.data[currentYear]) {
-			dataArray.push(graphData.data[currentYear][month]);
+		var dataArray = [], labelsArray = [], currentDate = new Date(),
+			currentYear = currentDate.getFullYear(), currentMonth = currentDate.getMonth();
+		// Get the data from the last year
+		for (month = currentMonth + 1; month <= 11; month++) {
+			if (!(currentYear - 1 in graphData.data)) {
+				dataArray.push(0);
+			} else {
+				dataArray.push(graphData.data[lastYear][month+1]);
```

Comment:
Where does this `lastYear` var come from?

Reply:
Ah sorry, my fault -- I removed the variable shortly before I pushed... will fix... THANKS!!!